To-do list/Controllers/Cell/TaskTableViewCell.swift

```diff
@@ -28,8 +28,9 @@ class TaskTableViewCell: UITableViewCell {
     
     func configure(_ task: Task) {
         titleLabel.text = task.title
-        descriptionLabel.text = task.description
-        deadlineLabel.text = task.deadline.formatDate()
+        contentLabel.text = task.content
+        deadlineLabel.text = task.deadline.convertToDateFormat(current: "yyyy-MM-dd HH:mm:ss",
```

Comment:
"yyyy-MM-dd HH:mm:ss" is used in more then 1 place within the app - make it global constant